### PR TITLE
V3s

### PR DESCRIPTION
**Pull request by Icenowy**
Add support for V3s SoC.

### CODE REVIEW

**Comment by n1tehawk** [comment]
Merged. Thanks!

File: soc_info.c [hunk 1]
```diff
@@ -171,6 +171,14 @@ soc_info_t soc_info_table[] = {
 		.swap_buffers = a10_a13_a20_sram_swap_buffers,
 		.sid_addr     = 0x01C14200,
 	},{
+		.soc_id       = 0x1681, /* Allwinner V3s */
+		.name         = "V3s",
+		.scratch_addr = 0x1000,
+		.mmu_tt_addr  = 0x8000,
```
Comment (ssvb) [inline soc_info.c]: Can you check/confirm that this is really needed? The output of the "sunxi-fel -v spl some-bootable-image" command should provide information about the initial state of the TTBR0 register.

**Comment by ssvb** [comment]
Thanks. Could you please also add V3s to this [wiki table](https://linux-sunxi.org/FEL/USBBoot#SoC_support_status)?

**Comment by Icenowy** [comment]
2016年12月28日 23:38于 Siarhei Siamashka <notifications@github.com>写道：
>
> @ssvb commented on this pull request.
>
> ________________________________
>
> In soc_info.c:
>
> > @@ -171,6 +171,14 @@ soc_info_t soc_info_table[] = {
>
>  .swap_buffers = a10_a13_a20_sram_swap_buffers,
>
>  .sid_addr     = 0x01C14200,
>
>  },{
>
> + .soc_id       = 0x1681, /* Allwinner V3s */
>
> + .name         = "V3s",
>
> + .scratch_addr = 0x1000,
>
> + .mmu_tt_addr  = 0x8000,
>
> Can you check/confirm that this is really needed? The output of the "sunxi-fel -v spl some-bootable-image" command should provide information about the initial state of the TTBR0 register.

MMU is not enabled by BROM

**Comment by ssvb** [comment]
> MMU is not enabled by BROM

OK, thanks! Do you have some information about the write speed improvement (with and without setting up the MMU from the sunxi-fel tool)? It is a part of the wiki table, so if you just work on populating this table with information, then all the necessary steps will be done.

**Pull request merged**